onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -220,6 +242,8 @@ def optimize_model(
     use_gpu: bool = False,
     only_onnxruntime: bool = False,
     verbose: bool = False,
+    *,
```

Comment:
Any reason to add * here?

Reply:
It is to add a new argument without its position. Those options will be used with their keywords anyway, IMHO, so I have added it as keyword-only.